tests/update_helper.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+#Stop on first error
+set -e
+
+#Echo all commands to terminal
+#set -x
+
+
+# Get the absolute path from a (potentially) relative path
+# $1 = relative path
+# Prints the absolute path to stdout for capture
+getAbsPath() {
+  local MY_PATH=$1
+  local ORIG_PATH=`pwd`
+
+  MY_PATH="`( cd \"$MY_PATH\" && pwd )`"
+  cd "$ORIG_PATH"
+
+  if [ -z "$MY_PATH" ] ; then
+    echo "Script path became something strange: '$SCRIPT_PATH'"
+    exit 1  # fail
+  fi
+  echo "$MY_PATH"
+}
+
+
+#Print usage if requested
+usage() {
+  #Grep through this source file for the options
+  echo "$0 usage:" && grep "    .)\ # " $0
+  exit 1
+}
+
+
+
+#Get the script absolute path to reference other scripts
+SCRIPT_PATH="`dirname \"$0\"`"
+SCRIPT_PATH=`getAbsPath "$SCRIPT_PATH"`
+
+
+
+#Get options
+OPM_DATA=
+WORKSPACE="$SCRIPT_PATH/.."
+
+if [ "$OSTYPE" == "linux-gnu" ]
+then
+  if which nproc > /dev/null
+  then
+    BUILDTHREADS=`nproc`
+  fi
+else # OSX
+  BUILDTHREADS=`sysctl hw.ncpu | awk -F ' ' '{print $1}'`
+fi
+test -z "$BUILDTHREADS" && BUILDTHREADS=1
+
+PULL_REQUESTS=
+[ $# -eq 0 ] && usage
+while getopts "d:w:p:t:o:h" arg; do
+  case $arg in
+    d) # OPM Data path to use
+      echo "OPM_DATA ${OPTARG}"
+      OPM_DATA=${OPTARG}
+      ;;
+    w) # Workspace directory to use for opm-simulators
+      echo "WORKSPACE ${OPTARG}"
+      WORKSPACE=${OPTARG}
+      ;;
+    t) # Build threads
+      echo "BUILDTHREADS ${OPTARG}"
+      BUILDTHREADS=${OPTARG}
+      ;;
+    p) # Select pull requests to build (e.g., opm-core=1100)
```

Comment:
it's a bit hard to do any kind of verification here as there is nothing to verify against per se. what do you suggest is done?